### ISSUE TITLE
Settlers instead of the upgraded units when starting a game in later eras

### ISSUE THREAD

**Issue by mt-empty** [LoneGazebo/Community-Patch-DLL#6142]
_1. Mod version (i.e Date - 4/23):_
Vox Populi (1-11-2)

_2. Mod list (if using Vox Populi only, leave blank):_

_3. Error description:_ 
Starting a game in industrial+ eras spawns setters rather than pioneers or colonists.

_4. Steps to reproduce (optional):_
Start a game in atomic era, the game will spawn 3 settlers, instead of 3 colonists
![1](https://user-images.githubusercontent.com/59728838/72681159-bdbe9a00-3ab8-11ea-9431-0aed72902df6.jpg)

**Comment by Stalker012345** [comment]
Even though they are settlers, the late game setting will make them spawn a whole bunch of buildings and such. So they are effectively colonists. I ask this a long time ago, and they said its a hardcoded part of the advanced era settings.

**Comment by mt-empty** [comment]
I started a game in industrial era, it definitely spawned some building when I settled but with no additional territory. 
I upgraded the other two settlers to pioneers and I got more building that the first settler.

![f (1)](https://user-images.githubusercontent.com/59728838/72691695-51c34c80-3b1f-11ea-8bbc-fe7263a75331.jpg)
![f (2)](https://user-images.githubusercontent.com/59728838/72691697-51c34c80-3b1f-11ea-9237-5b5999460782.jpg)
![f (3)](https://user-images.githubusercontent.com/59728838/72691699-52f47980-3b1f-11ea-8caa-fac680b2c42e.jpg)
It is even wore in atomic as the other two setters can be upgraded to colonist.
![f (4)](https://user-images.githubusercontent.com/59728838/72691700-538d1000-3b1f-11ea-8914-4e9b23d7d6e1.jpg)
